packages/js/src/components/WincherTableRow.js

```diff
@@ -132,6 +132,15 @@ export function getKeyphrasePosition( keyphrase ) {
 	return keyphrase.position.value;
 }
 
+/**
+ * Humanize the last updated date string
+ *
+ * @param {string} dateString The date string to format.
+ *
+ * @returns {string} The formatted last updated date.
+ */
+const formatLastUpdated = ( dateString ) => moment( dateString ).fromNow();
```

Comment:
I realized... Will this work even for other languages? Not sure on Yoast's policy on translations nor how it works with moment. Maybe it's auto-handled somehow.

Reply:
Seems to work fine and is related to wordpress admin language (settings --> General)

![image](https://github.com/wincher-ab/wordpress-seo/assets/129404933/ce842b4d-4598-48ec-96c3-a7a44259ed85)